Sming/Core/HardwarePWM.h

```diff
@@ -121,8 +124,21 @@ class HardwarePWM
 	/** @brief  This function is used to actually update the PWM.
 	 */
 	void update();
+	
+	#if SMING_ARCH == Esp32
+		uint32_t getFrequency(uint8_t pin);
+	#endif
```

Comment:
We should implement this for all architectures, so remove conditional.

NB. Use `#ifdef ARCH_ESP32`, there's no definition for 'Esp32'.